pkg/scaffold/internal/single-stack-sample/dev/kcl.mod

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "{{ .ProjectName }}"
-edition = "0.5.0"
+edition = "0.1.0"
```

Comment:
Does this edition represent the version of KCL? As far as I know, the KCL version we used is not 0.1.0

Reply:
do not need edition anymore